webapp/src/assets.ts

```diff
@@ -70,6 +70,13 @@ export function getAssets(gallery = false, firstType = pxt.AssetType.Image, temp
         }
     }
 
+    // sort so that non-builtin assets appear at the top of the gallery
```

Comment:
can we maybe combine this with the sorting we do for in `compareInternalId` -- that'd just basically be `return comparePackage if that value is non-zero, otherwise do current compareInternalId`. We shouldn't have to worry about the assets that get added in between that sort (L46-51) and here as those are all temporary assets which go into myAssets not the gallery, and combining the sorts is nice as it makes reasoning about the sorting easier (that way you can see the full sorting logic in one place & you don't have to know that javascript sort is guaranteed stable by default as of es2019)